ts/Core/Templating.ts

```diff
@@ -172,10 +174,17 @@ function dateFormat(
  * @param {Highcharts.Chart} [chart]
  *        A `Chart` instance used to get numberFormatter and time.
  *
+ * @param {Highcharts.Time} [time]
+ *        A `Time` instance used to get dateFormat.
+ *
  * @return {string}
  *         The formatted string.
  */
-function format(str = '', ctx: any, chart?: Chart): string {
+function format(
+    str = '',
+    ctx: any,
+    templateOwner?: Templating.TemplatingOwner
```

Comment:
`owner` is short and sufficient. The interface, `Templating.TemplatingOwner` has a redundant word, so it should probably be shortened to just `Templating.Owner`.